source/base/Profiler.ooc

```diff
@@ -65,6 +65,7 @@ Profiler: class {
 	}
 	resetAll: static func { This _profilers apply(func (profiler: This) { profiler reset() }) }
 	dispose: static func {
+		This _profilers[0] printResults()
```

Comment:
But this is not what I have in mind when I think of a `dispose()` method, though. @davidhesselbom ?